bin/node/Cargo.toml

```diff
@@ -16,56 +16,56 @@ targets = ["x86_64-unknown-linux-gnu"]
 name = "aleph-node"
 
 [dependencies]
-structopt = "0.3.8"
-log = "0.4"
-serde = "1"
-serde_json = "1"
-futures = "0.3"
-hex-literal = "0.3.1"
-libp2p = "0.39.1"
-hex = "0.4.2"
+clap = { version = "3.0", features = ["derive"] }
+log = "0.4.16"
+serde = "1.0.136"
```

Comment:
I would suggest specifying only major and minor version if major is >0, patch should almost always be irrelevant for our purposes, so including it is just confusing.

Reply:
Done!

Reply:
Ah, I was hoping you would do this in _all_ `Cargo.toml` files, not just this one. <_<"

Reply:
Will do, sir!

Reply:
Aaand done :)